org.alloytools.alloy.application/src/main/java/edu/mit/csail/sdg/alloy4viz/VizGUI.java

```diff
@@ -804,9 +804,7 @@ private void updateDisplay() {
                 scroll.addFocusListener(new FocusListener() {
 
                     @Override
-                    public final void focusGained(FocusEvent e) {
-                        t.requestFocusInWindow();
-                    }
+                    public final void focusGained(FocusEvent e) { }
```

Comment:
Why?

Reply:
Sorry, remnants of tests I was making. Cleared 🙆‍♂